packages/libs/contracts-sdk/package.json

```diff
@@ -15,11 +15,23 @@
   "publishConfig": {
     "access": "public"
   },
+  "scripts": {
+    "test": "jest"
+  },
   "devDependencies": {
+    "@dotenvx/dotenvx": "^1.44.2",
+    "@lit-protocol/auth-helpers": "^7.2.0",
+    "@lit-protocol/constants": "^7.2.0",
+    "@lit-protocol/lit-node-client": "^7.2.0",
     "@openzeppelin/contracts": "5.3.0",
-    "pino-pretty": "^13.0.0"
+    "@types/jest": "^29.5.12",
+    "jest": "^29.5.0",
+    "pino-pretty": "^13.0.0",
+    "ts-jest": "^29.1.0"
   },
   "dependencies": {
-    "@lit-protocol/logger": "custom-logger"
+    "@lit-protocol/logger": "custom-logger",
+    "@lit-protocol/pkp-ethers": "^7.2.0",
```

Comment:
These should be dev deps if they are only relevant to testing :+1: -- this is why NX is throwing a Lint error about them in the CI check; if they aren't used by any code inside the `build` output directory, then they are considered incorrectly placed to be in standard `dependencies` :+1:

@Ansonhkg What is the `custom-logger` tag of `@lit-protocol/logger` and why do we have it just for this package? 🤔

Reply:
:+1: `pkp-ethers` is only used as a dev dependency right now since we get the signer passed into the methods, so we can :hocho: the `custom-logger` dependency and move `@lit-protocol/pkp-ethers` to `devDependencies` @DashKash54